### PR TITLE
Upgrade deps to support babel 7

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = function(defaults) {
   if (includePolyfill) {
     babelOptions.includePolyfill = true;
   } else {
-    babelOptions.blacklist = ['regenerator'];
+    // babelOptions.exclude = ['regenerator'];
```
Comment (NullVoxPopuli) [inline ember-cli-build.js]: not sure what this would be for the new babel stuff

**Comment by machty** [comment]
What's the status of this?

**Comment by NullVoxPopuli** [comment]
@machty I got stuck and didn't know what I was doing, and this was old enough where I had forgotten about it. :-\

**Comment by MichalBryxi** [comment]
Actually I have a feeling that the lack of this one is now breaking my build:

```
  - originalErrorMessage: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.
  - stack: Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.
```

From all the packages I use this one seems the only one with 6.x.

**Comment by knownasilya** [comment]
Seems to work with ember-cli-babel 7.1.0 but not 7.4.0

**Comment by maxfierke** [comment]
@NullVoxPopuli Closing this out as master has been updated w/ ember-cli-babel 7. Thanks!

**Pull request closed without merging**